gitlab/README.md

```diff
@@ -166,16 +169,9 @@ Sample of built-in dashboard in SignalFx:
 
 ![Embedded PostgreSQL](./img/gitlab_embedded_postgresql_dashboard.png)
 
-#### Important conditions to watch out for
-
-Configuring GitLab by editing `/etc/gitlab/gitlab.rb` should be accompanied by running the command `gitlab-ctl reconfigure` in order for the changes to take effect.
-
-
-Nginx configuration by editing file `/var/opt/gitlab/nginx/conf/nginx-status.conf` for instance, should be accompanied by running command `gitlab-ctl restart`. Note that changes to the configuration file `/var/opt/gitlab/nginx/conf/nginx-status.conf` in particular are erased by subsequent runs of command `gitlab-ctl reconfigure` because `gitlab-ctl reconfigure` restores the original configuration file. 
-
 ### METRICS
 
-For documentation of the metrics and dimensions emitted:
+For documentation of the metrics and dimensions emitted per metrics endpoint:
```

Comment:
Please check with @asuresh4 for the standard language here. I think his script expects specific wording.